delta/plugins/elasticsearch/src/main/resources/defaults/default-settings.json

```diff
@@ -1,14 +1,45 @@
 {
   "analysis": {
     "analyzer": {
-      "nexus": {
+      "default": {
```

Comment:
As the default settings, may be used by custom views, we need to keep a reference to the nexus analyzer

Reply:
I've made the change. The `_createdAt` and `_updatedAt` use the `nexus` analyzer but they should probably be changed to use a pattern one because `1970-01-01T00:00:00Z` is tokenized as follows

```
1970-01-01t00, 1970, 01, 01t00, 00, 00z
```